test/remote/device_tests.js

```diff
@@ -378,7 +378,7 @@ describe('remote device', function () {
   it(
     'update device works with callback urls that have ports',
     () => {
-      var goodPushCallback = 'https://updates.push.services.mozilla.com:433'
+      var goodPushCallback = 'https://updates.push.services.mozilla.com:4430/';
```

Comment:
@shane-tomlinson we need to keep a test that has just the `:443`

Reply:
Ah nooo, that was a mistake.